retrofitadapter/src/main/java/cz/ackee/retrofitadapter/AckroutineCallAdapterFactory.kt

```diff
@@ -39,6 +40,6 @@ class AckroutineCallAdapterFactory(vararg interceptor: CallFactoryInterceptor) :
             BodyCallExecuteInterceptor()
         }
 
-        return AckroutineCallAdapter<Any>(responseType, interceptors + callInterceptor)
+        return AckroutineCallAdapter<Any>(responseType, annotations,interceptors + callInterceptor)
```

Comment:
weirdly placed comma, it seems that space is missing after that 🕵️

Reply:
✅